charts/argo-workflows/templates/controller/workflow-controller-servicemonitor.yaml

```diff
@@ -47,4 +49,5 @@ spec:
   selector:
     matchLabels:
       {{- include "argo-workflows.selectorLabels" (dict "context" . "name" .Values.controller.name) | nindent 6 }}
+#{{- end }}
```

Comment:
Maybe here should be fixed? 🙋 
https://github.com/argoproj/argo-helm/pull/2430#discussion_r1460697737

Reply:
addressed in f68d03c